bin/bundix

```diff
@@ -1,4 +1,15 @@
 #!/usr/bin/env ruby
 
-require 'bundix/cli'
-Bundix::CLI.start(ARGV, debug: true)
+require_relative '../lib/bundix'
+
+cli = Bundix::CLI.new(ARGV)
+command = ARGV.shift
+if cli.commands.include?(command)
+  cli.send("#{command}_options").order!
+  cli.send(command)
+elsif command # show help on unknown commands
+  puts cli.global.help
```

Comment:
Not a blocker, but might be nice to have a non-zero exit code when given an unknown command.